tests/functional/odf-cli/test_set_recovery_profile_to_favour_new_io.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import logging
+
+from ocs_ci.helpers.helpers import (
+    odf_cli_set_recover_profile,
+    get_ceph_recovery_profile,
+)
+from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.testlib import tier1
+
+log = logging.getLogger(__name__)
+
+
+@brown_squad
+@tier1
```

Comment:
are we setting this default and that why its part of tier1? because in cases where we need recovery IO to be faster good to set it based on that so good to add this part of tier4 or recovery scenario like node, osd failure tests.